_episodes/introduction.md

```diff
@@ -60,18 +59,19 @@ CC-BY licence. Source: <https://doi.org/10.5281/zenodo.3695300>*
 
 > ## What's in it for you?
 >
-> Many funders nowadays require data management plans in which the researchers
-> describe their adherence to the FAIR principles. A great step forward, but if 
+> Many funders nowadays require from researchers data management plans which
+> adhere to the FAIR principles. A great step forward, but if 
 > we are going to spend the time and effort, we prefer to find some intrinsic 
 > motivation.
 >
-> Have a look at the original [Nature paper]() on FAIR data, the [GO-FAIR
-> living document](https://www.go-fair.org/fair-principles/), and the Danish
-> tutorial [howtofair.dk](https://www.howtofair.dk/why-fair/). What are the main
-> reasons for *you* to (not) embrace the FAIR principles?
+> Have a look at the original [Nature paper](https://doi.org/10.1038/sdata.2016.18)
+> on FAIR Data, the [GO-FAIR initiative](https://www.go-fair.org/fair-principles/),
+> and the Danish tutorial [howtofair.dk](https://www.howtofair.dk/why-fair/).
+> What are the main reasons for *you* to (not) embrace the FAIR principles?
 {: .discussion}
 
-## FAIRification is a process
+## How FAIR are my data?
+_AK: The title was a bit misleading; there isn't much about FAIRification process itself._
```

Comment:
```suggestion
## How FAIR are my data?
```

Reply:
@stefsmeets this suggestion is made to the wrong line I think. Cannot commit it. Can you change it?